docs/docs/guides/receiving/filtering.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: default
+title: Subscriptions with filter
+nav_order: 2
+parent: Receiving messages
+grand_parent: How to Guides
+---
+
+When you receive messages from a subscription with a filter, you only receive the messages that match the filter. Metro automatically acknowledges the messages that don't match the filter. You can filter message by their attributes.
```

Comment:
> Metro automatically acknowledges the messages
Let's avoid internal implementation details in the documentation. Instead, can we say "Metro filters messages that match the filtering criteria before delivery"

Reply:
Done